src/routes/_layout.svelte

```diff
@@ -1,13 +1,10 @@
 <script>
   import Nav from "../components/Nav.svelte";
   import Footer from "../components/Footer.svelte";
-  import "../components/GlobalCSS.svelte";
-
-  export let segment;
 </script>
 
-<style>
-
+<style global src="../styles/global.css">
```

Comment:
Can we import it like `import "../styles/global.css";`? For webpack it would bundle the CSS as global styles, but I'm not sure if our tooling support this.

Reply:
This does work but it does not pipe it through postCSS and just bundles the unprocessed CSS.

Reply:
Is it possible to configure postCSS to process this?

Reply:
I'm trying to do this with `rollup-plugin-postcss`. Having a lot of unclear issues with the build at the moment though.

Reply:
That's fine, I think it's pretty clean already. If we can't get it working, we can just leave it be and continue with other tasks, and revisit it if we got time.